packages/lesswrong/components/editor/RateLimitWarning.tsx

```diff
@@ -11,9 +12,23 @@ const RateLimitWarning = ({lastRateLimitExpiry, rateLimitMessage}: {
   // "fromNow" makes for a more human readable "how long till I can comment/post?".
   // moment.relativeTimeThreshold ensures that it doesn't appreviate unhelpfully to "now"
   moment.relativeTimeThreshold('ss', 0);
-  const fromNow = moment(lastRateLimitExpiry).fromNow()
+  // format momentJS fromNow to say "3 seconds" or "3 minutes" rather than 3s or 3m
+  moment.updateLocale('en', {
+    relativeTime: {
+      s: 'a few seconds', ss: '%d seconds',
+      m: 'a minute',      mm: '%d minutes',
+      h: 'an hour',       hh: '%d hours',
+      d: 'a day',         dd: '%d days',
+      w: 'a week',        ww: '%d weeks'
+    }
+  });
```

Comment:
@Raemon FYI the reason I didn't go with this type of thing is that I think it is a [global change](https://momentjs.com/docs/#/customization/), and this might be stepping on the other place in the code where we define these

Reply:
So I was somewhat confused about that. It does seem like it *should* be a global change, but it doesn't affect the other instances of fromNow() or .from() that I checked. (i.e. it doesn't affect comment item dates or post item dates, even when rendered on the same page as a RateLimitWarning component).

Reply:
I just experimented a bit more, putting the relevant code in RateLimitWarning, FormatDate, and Layout. It seems to affect all children of the component where you run the code, which is convenient.